static/src/components/GameScreen.js

```diff
@@ -332,7 +332,7 @@ export class GameScreen extends Component {
         if (!started) {
             this.playersLogged();
         }
-        if (started && game.game.age) {
+        if (started && game) {
```

Comment:
I think this needs to be:
```
if (started && game) {
  if (game.game.age) {
    //
  }
}
```

Reply:
Hmmm it must've been messed with by all the commits and merging that have happened. Will sort now.